packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -365,55 +385,89 @@ const extractTemplate = (templateName, outputDir) => {
     sh.rm('-rf', tmp)
 }
 
-const main = (opts) => {
-    if (!(opts.outputDir === DEFAULT_OUTPUT_DIR) && sh.test('-e', opts.outputDir)) {
-        console.error(
-            `The output directory "${opts.outputDir}" already exists. Try, eg. ` +
-                `"~/Desktop/my-project" instead of "~/Desktop"`
-        )
-        process.exit(1)
-    }
-
-    switch (GENERATOR_PRESET) {
+const selectGenerator = (selectedOption, opts) => {
+    switch (selectedOption) {
         case HELLO_WORLD_TEST_PROJECT:
             return generateHelloWorld({projectId: 'hello-world'}, opts)
         case HELLO_WORLD:
             return helloWorldPrompts(opts).then((answers) => generateHelloWorld(answers, opts))
         case TEST_PROJECT:
             return runGenerator(testProjectAnswers(), opts)
-        case PROMPT:
+        case RETAIL_REACT_APP_DEMO:
+            return runGenerator(demoProjectAnswers(), opts)
+        case RETAIL_REACT_APP:
             console.log(
-                'See https://developer.salesforce.com/docs/commerce/commerce-api/guide/commerce-api-configuration-values.html for details on configuration values\n'
+                'For details on configuration values, see https://developer.salesforce.com/docs/commerce/commerce-api/guide/commerce-api-configuration-values\n'
             )
-            return prompts(opts).then((answers) => runGenerator(answers, opts))
+            return retailReactAppPrompts(opts).then((answers) => runGenerator(answers, opts))
+        case NO_SELECTION:
+            return selectorPrompt(opts)
```

Comment:
Let me show you how to do this and make it tasty please. This is a great idea!

Reply:
Sure, I'll await your input on this. Thanks!

Reply:
We paired on this. LGTM.